files-community/Files
# Fix: Fixed issue where progress was not updated when creating an archive

**Resolved / Related Issues**

**Archive Name**
- Operation banner was not reporting the name of the archive being created. This PR fixes that issue

**Archive Progress**
- This PR doesn't completely resolve the issue (I'll try to open a different PR in SevenZip repo to fix that). 
- If there are at least two folders, progress will be shown correctly (4 folders: 0% -> 25% -> 50% -> 75% -> Completed)

**Validation**
How did you test these changes?
- [ x Did you implement any design changes to an existing feature?
   1. Select at least two folders
   2. Create an archive
   3. See that progress is now being reported
